patches/tModLoader/Terraria/UI/ItemSorting.cs.patch

```diff
@@ -17,3 +17,20 @@
  				Item item = new Item();
  				item.netDefaults(i);
  				list2.Add(item);
+@@ -831,14 +_,14 @@
+ 		private static void SetupSortingPriorities() {
+ 			Player player = Main.player[Main.myPlayer];
+ 			_layerList.Clear();
+-			List<float> list = new List<float> {
++			List<Modifier> list = new List<Modifier> {
+ 				player.meleeDamage,
+ 				player.rangedDamage,
+ 				player.magicDamage,
+ 				player.minionDamage
+ 			};
+ 
+-			list.Sort((float x, float y) => y.CompareTo(x));
++			list.Sort((x, y) => ((float)y).CompareTo((float)x));
```

Comment:
unnecessary?

Reply:
It is necessary, that line replaces Comparison<float> with Comparison<Modifier>.